src/editor/components/graph/docs-dialog.tsx

```diff
@@ -0,0 +1,38 @@
+import { useEffect, useState } from 'react'
+import { Dialog } from '../dialog'
+import { Markdown } from '../markdown'
+
+interface DocsDialogProps {
+  areDocsOpen: boolean;
+  setAreDocsOpen: ( open: boolean ) => void;
+  filePath: string
+}
+
+export const DocsDialog = ( { areDocsOpen, setAreDocsOpen, filePath }: DocsDialogProps ): JSX.Element => {
+  const [content, setContent] = useState<string>( '*Loading…*' )
```

Comment:
A loading text is already but preferably we would've a spinner. But you can leave it for now.